src/App.js

```diff
@@ -1,5 +1,7 @@
 import "./App.css";
+import { Cart } from "./frontend/pages/cart/cart";
 import { Home } from "./frontend/pages/home/Home";
+import { Wishlist } from "./frontend/pages/wishlist/Wishlist";
 import Mockman from "mockman-js";
```

Comment:
try use use index for every folder, import from that, it makes code clean

Reply:
will do that. thanks